packages/redux-toolbelt/README.md

```diff
@@ -117,68 +125,83 @@ myAction(5, true)
 // }
 ```
 
+The `argMapper` function will receive the arguments passed to the action creator and return an object that will be merged along the action type to make the actual Redux action. The arguments are usually the values for the `payload` and `meta` properties, but they might be whatever is needed.
+
 #### Actions Defaults
-There are situations where you want to creates actions that has logical relations with each other with a prefix, or a common default metadata.
+
+There are situations where you want to create actions that have logical relations with each other with a prefix, or a common default metadata.
 You can do it like so:
+
 ```js
 const makeCounterAction = makeActionCreator.withDefaults({
-  prefix: "COUNTER/",
-  defaultMeta: {log: true}
-})
+  prefix: 'COUNTER/',
+  defaultMeta: { log: true },
+});
 
-const increase = makeCounterAction('INCREASE')
-const decrease = makeCounterAction('DECREASE')
+const increase = makeCounterAction('INCREASE');
+const decrease = makeCounterAction('DECREASE');
 
-increase()
+increase();
 // ==> { type: 'COUNTER/INCREASE', meta: {log: true} }
 
-decrease()
+decrease();
 // ==> { type: 'COUNTER/DECREASE', meta: {log: true} }
 ```
 
 ### `makeReducer()`
-Creates a reducer that handles action creator[s] created with `makeActionCreator`.
 
-- The first argument is `actionCreator[s]` and it can be one of the following:
-  - `actionCreator`
+Creates a reducer that handles action creator[s] created with `makeActionCreator` or follow its conventions ([FSA](https://github.com/redux-utilities/flux-standard-action) compliance plus action type naming convention).
+
+* The first argument is `actionCreator[s]` and it can be one of the following:
+
+  * `actionCreator`
+
     ```
```

Comment:
I suggest tagging code blocks, such as this one, with js, for \```js instead of \```

Reply:
@Satyam look at [this block of code example](https://github.com/Satyam/redux-toolbelt/tree/master/packages/redux-toolbelt#makereducer).

It's missing syntax highlighting, which can be added to every `js` code block